src/pages/docs/change-workflow.hbs

```diff
@@ -0,0 +1,107 @@
+      <ul class="drizzle-b-List drizzle-b-List--nested">
+        <li>Research issue titles must begin with [Research]</li>
+        <li>The results of your research should be shared with the core Spark team.</li>
+        <li>A Research task is complete when a decision has been made by the core team about the next steps.</li>
+        <li>If a Design, Build, or Document task is required, that issue should be created at this point.</li>
+      </ul>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h3 class="drizzle-b-h3">2. Design</h3>
+      <p>Design tasks apply to visual or interaction changes. These tasks include things like creating wireframes for a new component, sketching out a change to an interaction, or designing a new icon.</p>
+      <ul class="drizzle-b-List drizzle-b-List--nested">
+        <li>Design issue titles must begin with [Design]</li>
+        <li>Designs can be created in whatever software is appropriate, like Sketch, Illustrator, or Invision.</li>
+        <li>Design tasks must follow the <a class="drizzle-b-Link" href="/docs/contributions">Contribution Guide</a>.</li>
+        <li>Brand new design ideas should be shared with the broader design community for critique and to validate its usefulness across products.</li>
+        <li>Design work must be approved by Spark's design lead and should be reviewed by an engineer on the core team if the task will require engineering work.</li>
+        <li>Finished Design tasks need to be added to the <a class="drizzle-b-Link" href="https://github.com/sparkdesignsystem/spark-design-toolkit">design toolkit</a>. these changes should follow the Build task process outlined below.</li>
+        <li>If a Build or Document task is required and doesn't already exist, it should be created at this point.</li>
+      </ul>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h3 class="drizzle-b-h3">3. Build</h3>
+      <p>Build tasks apply to any changes made in code. These can be changes to components, to the Spark Docs site, or to any tooling that is part of the system.</p>
+      <ul class="drizzle-b-List drizzle-b-List--nested">
+        <li>Build issue titles must begin with [Build]</li>
+        <li>Code changes must follow the guidelines in <a class="drizzle-b-Link" href="/gettingstarted/developers">Getting Started - Developers</a> for code style.</li>
+        <li>Build tasks must follow the <a class="drizzle-b-Link" href="/docs/contributions">Contribution Guide</a>.</li>
+        <li>Code changes must be approved by a member of Spark's core team and should be reviewed by Spark's design lead when design or interaction are affected.</li>
+        <li>If a Document task is required and doesn't already exist, it should be created at this point.</li>
+      </ul>
+    </div>
+
+    <div class="drizzle-o-ContentGrouping">
+      <h4 class="drizzle-b-h4">Process</h4>
```

Comment:
PROCESS doesn't should up as bold weight. It makes it not look like a heading when viewed with all the other content. What do you think?